lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -208,6 +211,18 @@ def self.is_firebasecmd_supported?(cmd)
         true
       end
 
+      def self.post_notes(app_id, release_id, release_notes)
+        payload = { releaseNotes: { releaseNotes: release_notes } }
+        if release_notes.nil? || release_notes.empty?
+          UI.message("No Release notes passed in. Skipping this step")
```

Comment:
nit: maybe lowercase `Release` -> `release` and add a `.` after `Skipping this step`?